lib/auth.php

```diff
@@ -127,7 +127,8 @@ protected function _ldap($id,$pw) {
 			($info=ldap_get_entries($dc,$result)) &&
 			@ldap_bind($dc,$info[0]['dn'],$pw) &&
 			@ldap_close($dc)) {
-			return $info[0][$this->args['uid']][0]==$id;
+			$uid = strtolower($this->args['uid']);
+			return (mb_strtolower($info[0][$uid][0])) == (mb_strtolower($id));
```

Comment:
Fat-Free Framework tries to keep the "dependencies" as minimal as possible. The pull request introduces a dependency for the `mbstring` extension.

For the record: The system variable `ENCODING` defines the `mb_internal_encoding` when the extension `mbstring` is available. `mbstring` is an optional dependency at the moment.